workshop/README.md

```diff
@@ -32,31 +18,30 @@ The `Introduction` page has the following sections:
 ## Pre-requirements
 
 For this workshop you must have:
-* Docker CLI,
-* Docker Engine,
-* Docker Registry account,
+
+- Docker CLI,
+- Docker Engine,
+- Docker Registry account,
 
 ## Compatibility
 
 This workshop has been tested on the following platforms:
-* [Play with Docker](https://labs.play-with-docker.com/),
-* **Docker Desktop**: v2.2.0.5
-  * **Engine**: v19.03.8
-  * **Compose**: v1.25.4
+
+- [Play with Docker](https://labs.play-with-docker.com/),
+- **Docker Desktop**: v2.2.0.5
+  - **Engine**: v19.03.8
+  - **Compose**: v1.25.4
 
 ## Technology Used
 
-* [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/),
-* [Docker Hub](https://hub.docker.com/).
+- [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/),
+- [Docker Hub](https://hub.docker.com/).
 
 ## Credits
 
-* [John Zaccone](https://github.com/jzaccone)
-* [Jason Kennedy](https://github.com/jkomg)
-* [Steve Martinelli]()
-* [Remko de Knikker](https://github.com/remkohdev)
-
-## Translations
-
-* [Japanese](README-ja.md) - Thanks to [Toshio Yamashita](https://github.com/developerWorksJP)
+- [John Zaccone](https://github.com/jzaccone)
+- [Jason Kennedy](https://github.com/jkomg)
+- [Steve Martinelli](https://github.com/stevemar)
+- [Remko de Knikker](https://github.com/remkohdev)
 
+en
```

Comment:
not sure what this is doing here?

Reply:
it shouldn't, i used it to differentiate between 2 files when I was trying to debug the multi-language support, but had to remove multi-language support cause the travis setup doesnt support it i think and the gitbook layout was quite confusing and ugly.